### DISCUSSION TITLE
Creating migrations from a template

### DISCUSSION THREAD

**Discussion by jprokay** [amacneil/dbmate#528] [Feature Ideas]
:wave: I am a big fan of the simplicity of dbmate, but I do miss one feature from Sqitch: [Templates](https://sqitch.org/docs/manual/sqitch-add/#templates)
It is incredibly convenient to have templates, especially when working with a team, to ensure uniformity of migrations

I created a [PR to add support for templates](https://github.com/jprokay/dbmate/pull/1)
I haven't tested the crazy use cases that are possible with the `text/template` library, but for my use cases, it works great.

**Comment by dossy** [comment]
@jprokay Do you want to submit the pull request to the [amacneil:main](https://github.com/amacneil/dbmate/tree/main) branch?

**Reply by jprokay** [reply]
Yeah, I will do that. Thanks!

**Reply by dossy** [reply]
https://github.com/amacneil/dbmate/pull/530
